src/trigger.py

```diff
@@ -16,12 +15,18 @@
 import kernelci.config
 import kernelci.data
 from kernelci.cli import Args, Command, parse_opts
+from kernelci.data import kernelci_api
```

Comment:
This won't import the local `data/kernelci_api.py` file, so I don't think this is going to work.  We either need to use a local function, or use `db.something()` with a new method added in `kernelci-core` as that's where the `db` object is coming from.

For an early prototype, we can use a local function to try things out.  Then when it's working, we can add support for this in `kernelci.data.Database` with a new method to retrieve database entries and use it instead.

Reply:
Yes, I'll move the function back to trigger.py. Once this gets merged, we'll move it to `kernelci-core`.

Reply:
Thanks, sorry if my previous review wasn't clear enough.